documentation/manual/src/main/asciidoc/en-US/modules/mongodb.asciidoc

```diff
@@ -3254,32 +3256,68 @@ No cursor operations such as `sort()` are supported.
 Instead use the corresponding MongoDB http://docs.mongodb.org/manual/reference/operator/query-modifier/[query modifiers]
 such as `$orderby` within the criteria parameter.
 
+You can limit the results of a query using the `setMaxResults(...)` method.
+
 JSON parameters passed via the CLI syntax must be specified using the
 http://docs.mongodb.org/manual/reference/mongodb-extended-json/[strict mode].
 Specifically, keys need to be given within quotes; the only relaxation of this is that single quotes
 may be used when specifying attribute names/values to facilitate embedding queries within
 Java strings.
 
+Note that results of projections are returned as retrieved from the MongoDB driver at the moment and
```

Comment:
Is there a JIRA issue for this already? Could you create one if not, and reference it here?